launch_chffrplus.sh

```diff
@@ -101,7 +101,6 @@ function tici_init {
   sudo su -c 'echo "performance" > /sys/class/devfreq/soc:qcom,memlat-cpu0/governor'
   sudo su -c 'echo "performance" > /sys/class/devfreq/soc:qcom,memlat-cpu4/governor'
   nmcli connection modify --temporary lte gsm.auto-config yes
```

Comment:
This is already set in the OS as well, remove?

Reply:
Doesn't seem to work from the OS. Try removing it from launch, and read back the property.

Reply:
I restarted without launching launch_openpilot and auto-config was true for me. I'll make sure with a clean reflash though

Reply:
It reverted to `no` after I deleted the `nmconnection` file in `/data`, I'll leave it in